mattermost-community/README.md

```diff
@@ -0,0 +1,41 @@
+# What is this?
+
+This is an example [opta](https://github.com/run-x/opta) configuration file to deploy Mattermost (Community Edition) on AWS.
+
+You can also use this button to fill in the variable fields in the config.
+[![Deploy](https://raw.githubusercontent.com/run-x/opta/main/assets/deploy-to-aws-button.svg)](https://app.runx.dev/deploy-with-aws?url=https%3A%2F%2Fgithub.com%2Frun-x%2Fopta-examples%2Fblob%2Fmain%2Fmattermost-community%2Fmattermost_aws.yaml&name=)
+
+# What does this do?
+It deploys a single container version of Mattermost (Community Edition) on EKS in AWS. It uses AWS managed the RDS instance. It also sets up various other resources like VPCs, subnets, load balancers etc.
+
+# Steps to deploy
+* Fill in the required variables in the config file
+* Run `opta apply -c mattermost_aws.yaml` on the config file
+
+That's it. Mattermost is deployed on AWS. You can find the AWS load balancer URL to access the deployment by running `opta output`
```

Comment:
just for the folks who have no previous knowledge of opta: mention the field name to look for `load_balancer_raw_dns` and the port to use